build-tools/build.sh

```diff
@@ -0,0 +1,56 @@
+#
+# Copyright (c) 2020 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+#!/bin/sh
+
+buildToolsFolder="$(basename "$(dirname "$0")")"
```

Comment:
I think we can just use `"$(dirname "$0")"` then buildToolsFolder will store the absolute path of build tools folder instead of relative path of build tools folder, using absolute path can ensure user can run the build script anywhere. (relative path only support user run build script in registry-support repo).

Reply:
Updated.